vm/src/stdlib/signal.rs

```diff
@@ -23,51 +23,64 @@ const SIG_IGN: libc::sighandler_t = 1;
 #[cfg(windows)]
 const SIG_ERR: libc::sighandler_t = !0;
 
-const NSIG: usize = 64;
+pub const NSIG: usize = 64;
 
 // We cannot use the NSIG const in the arr macro. This will fail compilation if NSIG is different.
-static mut TRIGGERS: [AtomicBool; NSIG] = arr![AtomicBool::new(false); 64];
+static TRIGGERS: [AtomicBool; NSIG] = arr![AtomicBool::new(false); 64];
 
 extern "C" fn run_signal(signum: i32) {
-    unsafe {
-        TRIGGERS[signum as usize].store(true, Ordering::Relaxed);
-    }
+    TRIGGERS[signum as usize].store(true, Ordering::Relaxed);
```

Comment:
Nice one! The `AtomicBool` appears the be mutable itself!

Reply:
Btw, did you run benchmarks on this change? This change might trigger a performance boost :)